letsdrill.rb

```diff
@@ -1,12 +1,30 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+  case integer
+  when 0..59
+    p "F"
```

Comment:
You shouldn't need to `p` print everything to return it, when you run the `letsdrill.rb` file it should have a set of expectations at the bottom that will output true or false.  Try to remove all the `p` and get this working :)